### PR TITLE
Add routing

### PR DESCRIPTION
**Pull request by AshleeTan**

### CODE REVIEW

File: src/components/Routing.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  BrowserRouter as Router,
+  Switch,
+  Route
+} from "react-router-dom";
+
+import SearchResults from "./SearchResults";
+import PropertyDetail from "./PropertyDetail";
+
+const Routing = () => {
+  return(
```
Comment (JakeLin) [inline src/components/Routing.js:11]: better to add a space after `return`

File: src/components/PropertyDetail.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { useParams } from "react-router-dom";
+
+const PropertyDetail = () => {
+  const { propertyId } = useParams();
+  return(
```
Comment (JakeLin) [inline src/components/PropertyDetail.js:5]: Please add a space after `return`

**Review by JakeLin — APPROVED** [review]
LGTM with a couple of minor comments, feel free to merge it.

File: src/components/Routing.js (hunk 1 above)
Comment (AshleeTan) [reply src/components/Routing.js]: fixed, thanks for your comment

**Pull request merged**